locale/pt-br/texts.po

```diff
@@ -57,6 +428,320 @@ msgstr "Saldo Disponível"
 msgid "Locked"
 msgstr "Bloqueado"
 
+#: src/screens/PaymentRequestDetail.js:69
+#, javascript-format
+msgid "You've just received **${ amount } ${ symbol }**"
+msgstr "Você recebeu **${ amount } ${ symbol }**"
+
+#: src/screens/PaymentRequestDetail.js:86
+msgid "PAYMENT REQUEST"
+msgstr "REQUISIÇÃO DE PAGAMENTO"
+
+#: src/components/TxDetailsModal.js:50 src/screens/PaymentRequestDetail.js:104
+msgid "Token"
+msgstr "Token"
+
+#: src/screens/PaymentRequestDetail.js:116 src/screens/Receive.js:29
+msgid "My Address"
+msgstr "Meu endereço"
+
+#: src/screens/PaymentRequestDetail.js:120
+msgid "Status"
+msgstr "Status"
+
+#: src/screens/PaymentRequestDetail.js:121
+msgid "Received"
+msgstr "Recebido"
+
+#: src/screens/PaymentRequestDetail.js:121
+msgid "Waiting confirmation"
+msgstr "Aguardando confirmação"
+
 #: src/screens/PinScreen.js:157
 msgid "Incorrect PIN Code. Try again."
 msgstr "PIN incorreto. Tente novamente."
+
+#: src/screens/PinScreen.js:53
+msgid "Enter your PIN Code "
+msgstr "Entre seu PIN"
```

Comment:
I think I prefer "Digite seu PIN" in all strings "Entre com o PIN..", what do you think?

Reply:
Done.